full/Dockerfile

```diff
@@ -38,7 +38,7 @@ RUN pip3 install ledgerwallet==0.4.0
 RUN apk del python_build_deps
 
 # Add cargo ledger (needs a version of Rust >= 1.70)
-RUN cargo +$RUST_STABLE_VERSION install --version 1.2.2 cargo-ledger
+RUN cargo +$RUST_STABLE_VERSION install cargo-ledger
```

Comment:
Doing this will result in less predictability regarding versions used. We won't be able to be sure that if we run two builds of the image file, the same version of cargo-ledger will be installed for each build.

Reply:
Ok but it is a little bit cumbersome to update our builders for every release of cargo-ledger. Moreover, cargo-ledger is mainly a wrapper of cargo build and should not impact the resulting binaries: two binaries generated from two different cargo-ledger versions should ideally be the same.